API/Controllers/SeatsController.cs

```diff
@@ -31,11 +31,27 @@ public IEnumerable<Seats> GetAllSeats()
         //    return _seatsBLL.GetSeatById(id);
         //}
 
-        // list me all the free bus seats
+        // list me all the free/reserved bus seats
         [HttpGet("{id}/{status}")]
         public IEnumerable<Seats> GetSeatsByBus(int id, string status)
         {
             return _seatsBLL.GetSeatsByBus(id, status);
         }
+
+        [HttpPost]
+        public IActionResult Index([FromBodyAttribute] Seats seat)
+        {
+            _seatsBLL.Insert(seat);
+            return Ok();
+        }
+
+        [HttpPut("{id}")]
+        public IActionResult Update([FromBodyAttribute] int id)
```

Comment:
is this method really updating the existing seat?  and what changes are actually updated? 
put should look something like this: 
[HttpPut("{id}")]
public IActionResult Update(int id, [FromBody] Seats updatedSeat)
{
   _seatsBLL.Update(id, updatedSeat);
   return Ok();
}

then the BLL should get the existing seat from db (DAL)  and then change it with the values from updatedSeat  and then save to db